config/shapes/file_names.py

```diff
@@ -844,8 +808,20 @@
 
                 ],
             },
-            
             **common_files_2016preVFP
         ),
     },
 }
+
+
+def remove_empty_lists(d: dict) -> dict:
+    for key in list(d.keys()):
+        value = d[key]
+        if isinstance(value, dict):
+            remove_empty_lists(value)
+        if isinstance(value, list) and not value:
+            del d[key]
+    return d
```

Comment:
I suggest to call this function `remove_empty_processes` to make it clearer what is removed.